flex-model/index.html

```diff
@@ -0,0 +1,378 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Pixel-Perfect</title>
+    <link rel="stylesheet" href="./styles/reset.css" />
+    <link rel="stylesheet" href="./styles/fonts.css" />
+    <link rel="stylesheet" href="./styles/style.css" />
+  </head>
+  <body>
+    <!-- Header Start -->
+    <header class="header">
+      <section class="navigation-wrapper content-container">
+        <!-- Logo Start -->
+        <a href="#" class="logo">
+          <img class="logo-icon" src="./images/logo.png" alt="logo-plane" />
+          <div class="logo-name">
+            <p class="main-logo-name">Site <span class="main-logo-name-esp">Name</span></p>
+            <p class="logo-caption">your caption here</p>
+          </div>
+        </a>
+        <!-- Logo End-->
+        <!-- Navigation Start -->
+        <nav class="navigation" id="popup-menu">
+          <input class="menu" type="checkbox" id="checkbox" />
+          <label class="navigation-menu-btn" for="checkbox">Menu</label>
+          <ul class="navigation-list">
+            <li class="navigation-item"><a href="#" class="navigation-link">Home</a></li>
+            <li class="navigation-item"><a href="#" class="navigation-link">About</a></li>
+            <li class="navigation-item"><a href="#" class="navigation-link">Services</a></li>
+            <li class="navigation-item"><a href="#" class="navigation-link">Portfolio</a></li>
+            <li class="navigation-item"><a href="#" class="navigation-link">Contact</a></li>
+          </ul>
+        </nav>
```

Comment:
Would have made all the opening button for the block. Readers can read the screen correctly.

```html
<input class="menu" type="checkbox" id="checkbox" />
<label class="navigation-menu-btn" for="checkbox">Menu</label>
<nav class="navigation" id="popup-menu">        
  <ul class="navigation-list">
    <li class="navigation-item"><a href="#" class="navigation-link">Home</a></li>
    <li class="navigation-item"><a href="#" class="navigation-link">About</a></li>
    <li class="navigation-item"><a href="#" class="navigation-link">Services</a></li>
    <li class="navigation-item"><a href="#" class="navigation-link">Portfolio</a></li>
    <li class="navigation-item"><a href="#" class="navigation-link">Contact</a></li>
  </ul>
</nav>
```